server/test/routes/licenses.test.js

```diff
@@ -0,0 +1,26 @@
+import { test } from 'node:test';
+import * as assert from 'node:assert';
+import { build } from '../helper.js';
+
+
+test('License API returns valid results', async (t) => {
+  const app = await build(t);
+
+  const res = await app.inject({
+    url: '/api/v1/licenses?license=P39332'
+  });
+  const { name, licenseNumber } = JSON.parse(res.payload);
+  assert.equal(name, 'Koo, Chih Ren Nicholas');
+  assert.equal(licenseNumber, 'P39332');
+})
+
+test('License API returns error for no matching results', async (t) => {
+  const app = await build(t);
+
+  const res = await app.inject({
+    url: '/api/v1/licenses?license=1'
+  });
+  const { message } = JSON.parse(res.body);
+  assert.equal(res.statusCode, 500);
```

Comment:
HTTP 500 is Internal Server Error, not sure I'd use this code for the response.

Can we return a HTTP 404 Not Found code instead...?

Reply:
Made the update so a license with no matches returns a `404` error and updated the test case to reflect this.